modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/service/test/DDMFormInstanceReportLocalServiceTest.java

```diff
@@ -125,6 +126,42 @@ public void testDeleteFormInstanceReport() throws Exception {
 			getFormInstanceReportByFormInstanceId(
 				_ddmFormInstance.getFormInstanceId());
 	}
+
+	@Test
+	public void testProcessNonexistingFormInstanceRecordVersion()
+		throws Exception {
+
+		DDMStructure ddmStructure = addStructure(
+			0, _classNameId, null, "Test Structure", null,
+			read("ddm-structure-radio-field.xsd"), StorageType.JSON.getValue(),
+			DDMStructureConstants.TYPE_DEFAULT);
```

Comment:
@marcosapmf, please don't create a `DDMStructure` using a **xsd** file as an input, because this is an old way to do it, instead use one of the methods available on [DDMStructureTestUtil](https://github.com/liferay/liferay-portal/blob/master/modules/apps/dynamic-data-mapping/dynamic-data-mapping-test-util/src/main/java/com/liferay/dynamic/data/mapping/test/util/DDMStructureTestUtil.java). 😉